voorhoede/tudelft-spacefinder
# Feat/replace markers with new markers

# Changes


- replace old markers with new markers

# Associated issue


Resolves: https://trello.com/c/HUfjkBbT/86-find-a-new-way-to-show-markers-on-the-map

# How to test


1. Open preview link
2. Check the new occupancy markers on the map

# Checklist
